packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -92,30 +92,31 @@ describe(`request-queue`, function() {
   });
 
   describe(`push method`, function() {
-    it(`should push the Request given in the private array`, async function() {
+    it(`should push the given Request in the private array`, async function() {
       callbacks.requestWillEnqueue = sinon.spy();
-
+      const spy = sinon.spy(queue, 'getQueue');
```

Comment:
I think you want to use the spy to check it was called `expect(spy.callCount).to.equal(1)` at the end of this test.